### PR TITLE
feat/adding payments by memberId API

### PR DESCRIPTION
**Pull request by AndriP8**
changes

- adding payments path for
- adding methods `GET` and `PUT` for payments path
- adding `GetPaymentListResponse`, `paymentByMemberId` and `paymentRequest` schema

### CODE REVIEW

File: api.json [hunk 1]
```diff
@@ -268,6 +349,27 @@
...
       },
+      "GetPaymentListResponse": {
+        "type": "object",
+        "required": ["data", "status", "message"],
+        "properties": {
+          "data": {
+            "type": "array",
+            "x-go-custom-tag": "bson:\"data\"",
+            "items": {
+              "$ref": "#/components/paymentByMemberId"
```
Comment (mnindrazaka) [inline api.json:360]: Ini kalau nama componentnya jadi `Payment` aja gimana ya ?

```suggestion
              "$ref": "#/components/Payment"
```

File: api.json [hunk 2]
```diff
@@ -471,13 +551,41 @@
           }
         }
       },
+      "paymentByMemberId": {
```
Comment (mnindrazaka) [inline api.json:554]: Lanjutan dari comment di atas, yang disini kalau diubah jadi `Payment` aja gimana ?

```suggestion
      "Payment": {
```

**Review by mnindrazaka — DISMISSED** [review]
LGTM

**Review by dannycahyo — APPROVED** [review]

**Review by mnindrazaka — APPROVED** [review]
LGTM

**Pull request merged**